ament_virtualenv/ament_virtualenv/install.py

```diff
@@ -148,7 +148,7 @@ def install_venv(install_base, package_name, python_version='3'):
             root_dir=venv_install_dir,
             python_version=python_version,
             requirements_filename=generated_requirements,
-            use_system_packages=False,
+            use_system_packages=True,
```

Comment:
in catkin_virtualenv, this is driven via cmake https://github.com/locusrobotics/catkin_virtualenv?tab=readme-ov-file#additional-cmake-options

Reply:
We could do something similar... except my new world order doesn't lend itself nicely to passing arguments with pure python packages. I made everything "automatic" by defining the install class in ament_virtualenv...

Reply:
@paulbovbel I figured out how we can supply options within setup.py like we do with catkin_virtualenv. This is taken from the new commit description:

```
Extra options are now possible, similar to catkin_virtualenv by
defining an "ament_virtualenv" dictionary within the options
argument to setup().

setup(
    # ...
    options={
        "ament_virtualenv": {
            "use_system_packages": True,
            "python_version": "3.10"
        }
    }
}
```
For the cmake case I think it would be no different than catkin_virtualenv, just provide the options to the cmake function.